meta/meta.go

```diff
@@ -1016,36 +1015,11 @@ func (m *Meta) GetHistoryDDLJob(id int64) (*model.Job, error) {
 	return job, errors.Trace(err)
 }
 
-// GetAllHistoryDDLJobs gets all history DDL jobs.
-func (m *Meta) GetAllHistoryDDLJobs() ([]*model.Job, error) {
-	pairs, err := m.txn.HGetAll(mDDLJobHistoryKey)
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	jobs, err := decodeJob(pairs)
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	// sort job.
-	sorter := &jobsSorter{jobs: jobs}
-	sort.Sort(sorter)
```

Comment:
In the old code. jobs will be sorted by the ID. But I don't know the NewHashReverseIter whether to sort them.

Reply:
it will, but it is in reverse order, I checked the code, and the order does not matter

Reply:
it is the same as before now.